workflow_overview.md

```diff
@@ -0,0 +1,57 @@
+
+# 📄 General Overview of Project Workflow
+
+Once your [onboarding](https://intranet.neuro.polymtl.ca/onboarding/README.html) is complete, you will be ready to tackle your project!
+
+## 🖥️ Setting up 🖥️
+
+**Step 1.**
+* Make sure that your VPN connection is established or that you are connected to the Polytechnique wifi.
+
+**Step 2.**
+* Log in to one of the available [Neuropoly compute nodes](https://intranet.neuro.polymtl.ca/computing-resources/neuropoly/README.html):
+```
+ssh <POLYGRAMES_USERNAME>@<STATION>.neuro.polymtl.ca
+```
+
+**Step 3.**
+* Create your project working directory:
+```
+cd data_nvme_<POLYGRAMES_USERNAME>
+mkdir <PROJECT_NAME>
+cd <PROJECT_NAME>
+```
+
+**Step 4. Developing version-controlled software**
+* Ideally, you are working on code in Github repository (either a branch of an existing repo, or a new one that you created).
+* After adding your NeuroPoly workstation [SSH key to your Github account](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account?platform=linux), you are ready to make a local fork of that remote repository:
+```
+cd data_nvme_<POLYGRAMES_USERNAME>/<PROJECT_NAME>
+git clone -b "<YOUR_WORKING_BRANCH>" git@github.com:<REPOSITORY>.git
+```
+
+**Step 5. The data**
+* It is critical to make sure that you know what data you are working with. 
+* Ideally, it should be in [BIDS](https://bids-specification.readthedocs.io/en/stable/) format on the [`data.neuro`](https://intranet.neuro.polymtl.ca/data/git-datasets.html) storage node: `data.neuro:datasets/<PROJECT_DATASET>`.
+* Thanks to `git annex`, the following command will copy the directory structure and some small files of your dataset on `data.neuro`:
+```
+cd data_nvme_<POLYGRAMES_USERNAME>/<PROJECT_NAME>
+git clone git@data.neuro.polymtl.ca:datasets/<PROJECT_DATASET> 
+```
+
+## 🌊 Workflow 🌊
+
+### ⌨️ Code
+Any changes you make to the code should be added in small commits and pushed to your github branch.
+
+### 💿 Data
+* If you need to access your data files directly, you can use `git annex` to download the larger files to the [Neuropoly computer](https://intranet.neuro.polymtl.ca/computing-resources/neuropoly/README.html) you are working from:
+```
+cd data_nvme_<POLYGRAMES_USERNAME>/<PROJECT_NAME>/<PROJECT_DATASET> 
+git annex get .
+```
+* However, in order to save space, make sure to "undownload" those big files once you are done working with them with:
+```
+git annex drop .
+```
```

Comment:
This is a good tip, but I think it would fit better as a new subsection over in https://intranet.neuro.polymtl.ca/data/git-datasets.html#drop.